.github/workflows/merge-dependabot.yml

```
@@ -0,0 +1,21 @@
+  workflow_run:
+    workflows:
+      - "Integration test"
+      - "Lint OpenAPI schema file with Spectral"
+    types:
+      - completed
+    conclusion: success
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.user.login == "dependabot[bot]"
```

actionlint (1.7.10)

[error] got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)